### PR TITLE
Readme and Manual Update

### PR DESCRIPTION
**Pull request by braunms**
Update of the MegaMol manual:

- Static build
- LUA project files
- New frontend `megamol`
- ImGui
- Built-in configurator
- Usage of Cinematic plugin with cinematic_editor.lua project

Links to 'rendered' markdown files:
[readme](https://github.com/braunms/megamol/blob/master/README.md)
[manual](https://github.com/braunms/megamol/blob/master/docs/manual.md)
[devguide](https://github.com/braunms/megamol/blob/master/docs/devguide/devguide.md)

### CODE REVIEW

File: docs/manual.md [hunk 1]
```diff
@@ -38,123 +40,157 @@ provided that the following conditions are met:
 
 THIS SOFTWARE IS PROVIDED BY THE MEGAMOL TEAM "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE MEGAMOL TEAM BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-<a name="installation-and-setup"></a>
 
+
+<!-- ###################################################################### -->
 ## Installation and Setup
+(Effective: 02.02.2021)
```
Comment (gralkapk) [inline docs/manual.md]: Please use ISO format ;)

File: docs/manual.md [hunk 2]
```diff
@@ -609,14 +587,14 @@ The input file name and output file name are explicitly specified using the -v a
     200|LRH Server socket closed
 
 To convert from other file formats, for which a corresponding loader does exist, you should be able to adjust this project file.
+--> 
 
-<a name="advanced-usage"></a>
-
+<!-- ###################################################################### -->
 ## Advanced Usage
```
Comment (gralkapk) [inline docs/manual.md:2675]: I think, we should move this to a different document.

File: docs/manual.md [hunk 3]
```diff
@@ -457,35 +434,37 @@ Some useful controls:
...
 
+<!-- ---------------------------------------------------------------------- -->
 ### Reproducibility
 
-MegaMol stores the active project and all parameter settings in the EXIF field of the saved screenshots. Please note that this field currently contains a simple zero-terminated string with the LUA code required to reproduce the state when the screenshot is taken, and **not** valid EXIF data. Such a project can be restored by just loading the PNG:
+MegaMol stores the active project and all parameter settings in the EXIF field of the saved screenshots. Please note that this field currently contains a simple zero-terminated string with the LUA code required to reproduce the state when the screenshot is taken, and **not** valid EXIF data. Such a project can be restored by just loading the PNG file:
+
 ```
-$ mmconsole -p something.png
+    $ mmconsole -p something.png
```
Comment (gralkapk) [inline docs/manual.md]: I think, we should not mention mmconsole.

File: docs/manual.md (hunk 1 above)
Comment (braunms) [reply docs/manual.md]: Deleted, since it is not neccessary ...

File: docs/manual.md (hunk 2 above)
Comment (braunms) [reply docs/manual.md:2675]: No more used/required ...  devguide can be used ...

File: docs/manual.md (hunk 3 above)
Comment (braunms) [reply docs/manual.md]: Fixed ...

File: docs/devguide/devguide.md [hunk 4]
```diff
@@ -108,9 +155,12 @@ It causes the graph updater to stop at the indicated event and delay further gra
...
 
+
+<!-- ###################################################################### -->
+-----
 ## Build System
 
-For building MegaMol, CMake is used. For developers, two aspects are of importance: adding new plugins, and [adding and using external dependencies](#external-dependencies).
+For building MegaMol, CMake is used. For developers, two aspects are of importance: [adding new plugins](#create-new-plugin), and [adding and using external dependencies](#external-dependencies).
 
 ### External dependencies
```
Comment (moritz-h) [inline docs/devguide/devguide.md:216]: All externals must be build static now. The installation of shared libraries was removed, therefore megamol.exe will not find the so/dll files of externals if they are used as shared library.

Comment (braunms) [reply docs/devguide/devguide.md:216]: Tackled in a9b78a8 ...

**Comment by braunms** [comment]
PR fixes issue  #689

**Comment by braunms** [comment]
Closing because git is confused ... better safe than sorry :-P

**Pull request closed without merging**